apps/rush-lib/src/api/CommandLineConfiguration.ts

```diff
@@ -41,6 +42,67 @@ export class CommandLineConfiguration {
     return new CommandLineConfiguration(commandLineJson);
   }
 
+  /**
+   * Loads the configuration from the specified file and applies any ommited default build
+   * settings.  If the file does not exist, then an empty default instance is returned.
+   * If the file contains errors, then an exception is thrown.
+   */
+  public static loadFromFileAndDefault(jsonFilename: string): CommandLineConfiguration {
+    let commandLineJson: ICommandLineJson | undefined = undefined;
+    if (FileSystem.exists(jsonFilename)) {
+      commandLineJson = JsonFile.load(jsonFilename);
+
+      // TODO: Investigate
+      const defaultBuildSettings: CommandJson[] =
+        [{
+          commandKind: 'bulk',
+          name: 'build',
+          summary: '(EXPERIMENTAL) Build all projects that haven\'t been built, or have changed since they were last'
+          + ' built.',
+          enableParallelism: true,
+          ignoreMissingScript: false,
+          ignoreDependencyOrder: false,
+          allowWarningsInSuccessfulBuild: false,
+          safeForSimultaneousRushProcesses: false
+        },
+        {
+          commandKind: 'bulk',
+          name: 'rebuild',
+          summary: 'Clean and rebuild the entire set of projects',
+          enableParallelism: true,
+          ignoreMissingScript: false,
+          ignoreDependencyOrder: false,
+          allowWarningsInSuccessfulBuild: false,
+          safeForSimultaneousRushProcesses: false
```

Comment:
shouldn't this also have incremental: true ?

Reply:
I've added incremental, but shouldn't it be, incremental: false, for default rebuild settings? Im basing this off the defaults defined by [_addDefaultBuildActions()](https://github.com/microsoft/rushstack/blob/fd7859b96485a097855efc6445f495784b3becbb/apps/rush-lib/src/cli/RushCommandLineParser.ts#L197-L221).